OpenLIFUHome/OpenLIFUHome.py

```diff
@@ -49,60 +57,11 @@ def __init__(self, parent):
             "and development."
         )
 
-        # Additional initialization step after application startup is complete
-        slicer.app.connect("startupCompleted()", registerSampleData)
-
-
 #
-# Register sample data sets in Sample Data module
+# General utilities that probably should go elsewhere?
+# TODO Move some of these functions to more suitable places, e.g. some need to be where all modules can access them.
```

Comment:
This section here contains a whole bunch of generically useful function, many of which should really be accessible to all modules in this extension. Where is a good place to put them?

Reply:
Let's follow up with @sjh26 but in the meantime this isn't a blocker for this PR.